caffe2/mobile/contrib/ios/ios_caffe_predictor.cc

```diff
@@ -50,8 +50,7 @@ Caffe2IOSPredictor::Caffe2IOSPredictor(const caffe2::NetDef& init_net,
 
 void Caffe2IOSPredictor::run(const Tensor& inData, Tensor& outData, std::string& errorMessage) {
   FLAGS_caffe2_force_shared_col_buffer = true;
-  caffe2::Tensor input(caffe2::CPU);
-  input.Resize(inData.dims);
+  caffe2::Tensor input(inData.dims, caffe2::CPU);
```

Comment:
Is there any particular reason we can't rewrite these in the factory function style, e.g.,

```
auto input = empty(inData.dims);
```

Reply:
will change to caffe2::empty